### PR TITLE
Speaker Announcement Post/Email

### PR DESCRIPTION
**Pull request by kelly-o**

### CODE REVIEW

File: docs/conf/portland/2018/news/announcing-speakers.rst [hunk 1]
```diff
@@ -0,0 +1,60 @@
+Happy Friday everybody! We're writing today with an update on the fast-approaching Portland conference!
```
Comment (thatdocslady) [inline docs/conf/portland/2018/news/announcing-speakers.rst]: para break between the sentences?

File: docs/conf/portland/2018/news/announcing-speakers.rst [hunk 2]
```diff
@@ -0,0 +1,60 @@
+Happy Friday everybody! We're writing today with an update on the fast-approaching Portland conference! 
+
+First off, we're delighted to announce that we've finalized our speakers for the event! Every year talk selection gets more and more difficult as we get a larger, stronger batch of submissions. Thank you so much to everyone who took the time to send us a pitch – we literally wouldn't have an event without you!
```
Comment (thatdocslady) [inline docs/conf/portland/2018/news/announcing-speakers.rst]: * we've > we (ooooooh what a nit pick)
* 1 sentence per line plz <3

File: docs/conf/portland/2018/news/announcing-speakers.rst [hunk 3]
```diff
@@ -0,0 +1,60 @@
+Happy Friday everybody! We're writing today with an update on the fast-approaching Portland conference! 
+
+First off, we're delighted to announce that we've finalized our speakers for the event! Every year talk selection gets more and more difficult as we get a larger, stronger batch of submissions. Thank you so much to everyone who took the time to send us a pitch – we literally wouldn't have an event without you! 
+
+We also have some info about ticket purchasing (now's the time!), trip planning (Portland's cool!), and our generous sponsors (want to join them?). Read on for more details! 
+
+Full speaker line-up
+--------------------
+Every year, we look to bring a wide range of voices to the Write the Docs stage. Because the role of "documentarian" looks so different to each of us, we aim for a line up that spans of good mix of practical, philosophical, and technical topics – with the odd whimsical one thrown in for good measure. We're really excited about the presentations we're going hear this year – we hope you are too!
```
Comment (thatdocslady) [inline docs/conf/portland/2018/news/announcing-speakers.rst]: * line up > line-up?
* spans of > spans?

File: docs/conf/portland/2018/news/announcing-speakers.rst [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+* Neal Kaplan – `Where do I start? The art and practice of documentation triage <http://www.writethedocs.org/conf/portland/2018/speakers/#speaker-portland-2018-neal-kaplan>`_
+* Sarah Day – `Starting from Scratch: Finding and Hiring Junior Writers <http://www.writethedocs.org/conf/portland/2018/speakers/#speaker-portland-2018-sarah-day>`_
+* Steve Stegelin – `Graphic Content Warning: The Pros, Cons, and Alternatives to Screenshots <http://www.writethedocs.org/conf/portland/2018/speakers/#speaker-portland-2018-steve-stegelin>`_
+* Ted Hudek – `Eight Essential Tips for the Enlightened Tech Writer <http://www.writethedocs.org/conf/portland/2018/speakers/#speaker-portland-2018-ted-hudek>`_
+* Thursday Bram – `What They Don't Tell You About Creating New Style Guides <http://www.writethedocs.org/conf/portland/2018/speakers/#speaker-portland-2018-thursday-bram>`_
+
+You can head over to the `speaker page <http://www.writethedocs.org/conf/portland/2018/speakers/>`_ to see the full abstracts. We hope there’s something there for everyone to enjoy, and also something that will stretch your horizons a little bit, too.
```
Comment (thatdocslady) [inline docs/conf/portland/2018/news/announcing-speakers.rst]: * speaker page > Speakers page
* "and also something... , too" sounds a bit repetitive. Maybe cut one of these?

File: docs/conf/portland/2018/news/announcing-speakers.rst [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+
+
+Tickets are going fast!
+-----------------------
+If you don't have your ticket yet, now's a great time to snap one up. The conference has sold out for the last several years, so if you're planning to come, don't leave it to the last minute.
```
Comment (thatdocslady) [inline docs/conf/portland/2018/news/announcing-speakers.rst]: "has sold out" sounds a bit like "omg you sellouts". Maybe "the conference tickets have sold out"?

File: docs/conf/portland/2018/news/announcing-speakers.rst [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+
+`Buy your tickets here! <http://www.writethedocs.org/conf/portland/2018/tickets/>`_
+
+Planning your trip
+------------------
+In the next few weeks, we’ll be announcing more details of the conference events: the full schedule, the annual hike on Saturday afternoon, the writing day and reception on Sunday, and the official party on Monday evening. But on top of that, there's ALL SORTS of stuff to do in Portland.
```
Comment (thatdocslady) [inline docs/conf/portland/2018/news/announcing-speakers.rst]: * details of > details about
* do we need all the "the"s in the list?

File: docs/conf/portland/2018/news/announcing-speakers.rst [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+-----------------------
+If you don't have your ticket yet, now's a great time to snap one up. The conference has sold out for the last several years, so if you're planning to come, don't leave it to the last minute.
+
+`Buy your tickets here! <http://www.writethedocs.org/conf/portland/2018/tickets/>`_
+
+Planning your trip
+------------------
+In the next few weeks, we’ll be announcing more details of the conference events: the full schedule, the annual hike on Saturday afternoon, the writing day and reception on Sunday, and the official party on Monday evening. But on top of that, there's ALL SORTS of stuff to do in Portland. 
+
+We've got a couple of excellent volunteers – Mo Nishiyama and Alicia Duncan – who are heading up our Explore Portland committee. They're already hard at work putting together resources to help you plan your visit! Check out the `Visit Portland <http://www.writethedocs.org/conf/portland/2018/visiting/>`_ page on the conference website, or pop into the `#wtd-conferences <https://writethedocs.slack.com/messages/wtd-conferences>`_ channel on Slack to ask a question.
```
Comment (thatdocslady) [inline docs/conf/portland/2018/news/announcing-speakers.rst]: link to volunteers' twitter accounts?

**Review by thatdocslady — COMMENTED** [review]
Love it! Bunch-o-nits ;)

**Review by ericholscher — APPROVED** [review]
Looks good to me! I hope you didn't generate the speaker list by hand, we have software that does that, so I'm hoping someone did it automatically!

File: docs/conf/portland/2018/news/announcing-speakers.rst [hunk 8]
```diff
@@ -0,0 +1,60 @@
+Happy Friday everybody! We're writing today with an update on the fast-approaching Portland conference! 
+
+First off, we're delighted to announce that we've finalized our speakers for the event! Every year talk selection gets more and more difficult as we get a larger, stronger batch of submissions. Thank you so much to everyone who took the time to send us a pitch – we literally wouldn't have an event without you! 
+
+We also have some info about ticket purchasing (now's the time!), trip planning (Portland's cool!), and our generous sponsors (want to join them?). Read on for more details!
```
Comment (plaindocs) [inline docs/conf/portland/2018/news/announcing-speakers.rst]: love it

**Review by plaindocs — APPROVED** [review]
Zero nits that haven't already been nitted.

File: docs/conf/portland/2018/news/announcing-speakers.rst (hunk 7 above)
Comment (kelly-o) [reply docs/conf/portland/2018/news/announcing-speakers.rst]: I don't have em handy at the moment, but I can add em in later.

File: docs/conf/portland/2018/news/announcing-speakers.rst (hunk 2 above)
Comment (kelly-o) [reply docs/conf/portland/2018/news/announcing-speakers.rst]: fixed – and re: line breaks, I'll try to get into the habit, but I'm making no promises.

**Pull request merged**